README.fr.md

```
@@ -13,171 +13,74 @@ MCPHub facilite la gestion et la mise à l'échelle de plusieurs serveurs MCP (M
-
-Où `{server}` est le nom du serveur auquel vous souhaitez vous connecter. Cela vous permet d'accéder directement à un serveur MCP spécifique.
-
-> **Note** : Si le nom du serveur et le nom du groupe sont identiques, le nom du groupe aura la priorité.
-
-### Point de terminaison SSE (obsolète à l'avenir)
-
-Connectez les clients IA (par exemple, Claude Desktop, Cursor, DeepChat, etc.) via :
-
-```
-http://localhost:3000/sse
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)